app-i18n/libhangul/libhangul-0.2.0.ebuild

```
@@ -0,0 +1,43 @@
+# Copyright 2006-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+inherit libtool
+
+DESCRIPTION="Library for hangul input method logic, hanja dictionary"
+HOMEPAGE="https://github.com/libhangul/libhangul"
+SRC_URI="https://github.com/libhangul/${PN}/releases/download/${P}/${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+SLOT="0/1"
+KEYWORDS="amd64 ~arm64 ~loong ppc ppc64 ~riscv x86"
```

You should drop all arches to `~arch` for version bumps; see [the devmanual](https://devmanual.gentoo.org/keywording/index.html#keywording-on-upgrades) for reasons.

```suggestion
KEYWORDS="~amd64 ~arm64 ~loong ~ppc ~ppc64 ~riscv ~x86"
```